packages/keystone/src/lib/core/graphql-errors.ts

```diff
@@ -27,6 +27,16 @@ export const extensionError = (extension: string, things: { error: Error; tag: s
   );
 };
 
+export const resolverError = (things: { error: Error; tag: string }[]) => {
+  const s = things.map(t => `  - ${t.tag}: ${t.error.message}`).join('\n');
+  return new ApolloError(
+    `An error occured while resolving input fields.\n${s}`,
+    'INTERNAL_SERVER_ERROR',
+    // Make the original stack traces available.
+    { debug: things.map(t => ({ stacktrace: t.error.stack, message: t.error.message })) }
```

Comment:
This is cleared out in prod by `formatError`, right?

Reply:
Yep.